examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -284,7 +288,8 @@ angular.module('cloudberry.common', [])
         }));
 
         var batchJson = (JSON.stringify({
```

Comment:
you may want to rename it as `batchWithGeoRequest` to be clear?

Reply:
I shall modify it.